### PR TITLE
Fix part of #1460: Json file updates for questions and skills

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR fixes part of #1460 

## Checklist
- [x] All questions are shifted to a single `questions.json` file.
- [x] All questions are shifted to a single `skills.json` file.

## Pending
- [ ] Structure of question json files needs to be updated. (Will be done in future PR)
- [ ] Topic Size Calculation.

## How to test:
1. All tests are passing without major change makes sure that the content is parsing correctly.
2. Run the app, go to `Fractions`-> `Practice`, start questions.
3. Make `ConceptCardFragmentTestActivity` as launcher activity and check that it works.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt [hunk 1]
```diff
@@ -844,76 +730,45 @@ class TopicController @Inject constructor(
     return workedExampleList
   }
 
-  private fun createTestConceptCardForSkill0(): ConceptCard {
-    return ConceptCard.newBuilder()
-      .setSkillId(TEST_SKILL_ID_0)
-      .setSkillDescription(createTestTopic0Skill0().description)
-      .setExplanation(
-        SubtitledHtml.newBuilder().setHtml("Hello. Welcome to Oppia.").setContentId(
-          TEST_SKILL_CONTENT_ID_0
```
Comment (anandwana001) [inline domain/src/main/java/org/oppia/domain/topic/TopicController.kt:853]: Can we remove the unused? Line number 50,51,56,57
```
const val TEST_SKILL_CONTENT_ID_0 = "test_skill_content_id_0"
const val TEST_SKILL_CONTENT_ID_1 = "test_skill_content_id_1"
.
.
.
const val TEST_QUESTION_ID_4 = "question_id_4"
const val TEST_QUESTION_ID_5 = "question_id_5"
```

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt [hunk 2]
```diff
@@ -821,16 +699,24 @@ class TopicController @Inject constructor(
...
+  private fun getSkillJsonObject(skillId: String): JSONObject {
+    val skillJsonArray = jsonAssetRetriever.loadJsonFromAsset("skills.json")?.optJSONArray("skills")
+      ?: return JSONObject("")
+    for (i in 0 until skillJsonArray.length()) {
+      val currentSkillJsonObject = skillJsonArray.optJSONObject(i)
+      if (skillId == currentSkillJsonObject.optString("id")) {
+        return currentSkillJsonObject
+      }
+    }
+    return return JSONObject("")
```
Comment (anandwana001) [inline domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: nit change, one return, else this will be unreachable code.

**Review by anandwana001 — APPROVED** [review]
LGTM, just some nit changes

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt (hunk 1 above)
Comment (rt4914) [reply domain/src/main/java/org/oppia/domain/topic/TopicController.kt:853]: Done.

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt (hunk 2 above)
Comment (rt4914) [reply domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: Took me a minute to realise this mistake. :-D Done.

Comment (anandwana001) [reply domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: Thanks :)

**Pull request merged**